plugins/leemons-plugin-tests/frontend/src/pages/private/tests/Result.js

```diff
@@ -368,7 +362,7 @@ export default function Result() {
   }
 
   // if (!store.room) {
-  if (store.isTeacher || (!store.isTeacher && store.feedback)) {
+  if (isTeacher || (!isTeacher && store.feedback)) {
```

Comment:
This condition is redundant:

isTeacher
or hasFeedback

Because the right-hand condition will only be evaluated if !isTeacher. So redundant.

Reply:
Resolved!